libs/positions/src/lib/use-positions-data.tsx

```diff
@@ -38,7 +38,8 @@ const getSummaryRow = (positions: Position[]) => {
 export const usePositionsData = (
   partyId: string,
   gridRef: RefObject<AgGridReact>,
-  assetSymbol?: string
+  assetSymbol?: string,
+  withSummaryRow?: boolean
```

Comment:
question (blocking): Do we need withSummaryRow? It looks like it never gets passed in